src/app/cube/details/action-panel/action-panel.component.ts

```diff
@@ -149,6 +150,17 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
     this.toaster.notify('Success!', 'Learning object link copied to your clipboard!', 'good', 'far fa-check');
   }
 
+  copyAttribution() {
+    const range = document.createRange();
+    range.selectNode(document.getElementById('objectAttribution'));
+    window.getSelection().removeAllRanges();
+    window.getSelection().addRange(range);
+    document.execCommand('copy');
+    window.getSelection().removeAllRanges();
+
+    this.toaster.notify('Success!', 'Learning object Creative Commons Attribution copied to your clipboard!', 'good', 'far fa-check');
```

Comment:
Learning Object should be capitalized:
```suggestion
    this.toaster.notify('Success!', 'Learning Object Creative Commons Attribution copied to your clipboard!', 'good', 'far fa-check');
```

But maybe we can simplify this a little more.
```suggestion
    this.toaster.notify('Success!', 'Attribution information has been copied to your clipboard!', 'good', 'far fa-check');
```